rebar.config

```diff
@@ -56,7 +56,11 @@
 
 %% == Dependencies ==
 
-{project_plugins, [{rebar3_lint, "0.2.0"}, rebar3_hex]}.
+{project_plugins, [
+    rebar3_lint,
```

Comment:
Moved to a more recent version to see if anything new surfaces. Hope this is Ok.